api/db/migrations/20211220105432_add-ownerId-to-campaigns.js

```diff
@@ -0,0 +1,16 @@
+const TABLE_NAME = 'campaigns';
+const COLUMN_NAME = 'ownerId';
+
+exports.up = function(knex) {
```

Comment:
Penser à rajouter les contraintes (si j'ai bien compris la PR):
- NOT NULL
- FK sur user.id

Reply:
Done ✅